subsys/net/lib/nrf_cloud/src/nrf_cloud_log_backend.c

```diff
@@ -286,7 +301,11 @@ static int logger_is_ready(const struct log_backend *const backend)
 			}
 		}
 	}
-
+	if (IS_ENABLED(CONFIG_NRF_CLOUD_COAP)) {
```

Comment:
does `if (IS_ENABLED(CONFIG_NRF_CLOUD_COAP) && !nrf_cloud_coap_is_connected()) {`
give a build error when not building for CoAP?

Reply:
Both the original and your change have trouble due to headers.  I think I've fixed it now, and took your change.